### ISSUE TITLE
Internal tags in media documents

### ISSUE THREAD

**Issue by josefspr** [daat-hamakom/data#76]
What are the possibilities for media documents with internal tags that refer to certain places inside the text ? Html documents have this possibility, I don't know about pdf.

**Comment by yuvadm** [comment]
Yes, for PDFs displayed in browsers you can use `http://example.com/file.pdf#page9`

**Comment by josefspr** [comment]
I don't understand the link, takes me to http://www.iana.org/domains/reserved

**Comment by yuvadm** [comment]
`#page9` is the internal anchor, try it on a real PDF link
